samples/createOccurrence.js

```diff
@@ -0,0 +1,34 @@
+// [START containeranalysis_create_occurrence]
+// Creates and returns a new Occurrence associated with an existing Note
+const createOccurence = async(
+    noteProjectId = 'your-project-id', // Your GCP Project Id
+    noteId = 'my-note-id', // Id of the note
+    occurenceProjectId = 'your-project-id', // GCP Project Id of Occurence
+    imageUrl = 'my-url.io/project/image' // Image to attach metadata to
```

Comment:
I started using GCR links for the image, since that's probably a more realistic use case. You might want to use something like "https://gcr.io/my-project/my-image:123"